rldb.py

```diff
@@ -0,0 +1,100 @@
+from os import path
+import sqlite3
+
+# Original Repository: https://github.com/eibex/reaction-light
+# License: MIT - Copyright 2019-2020 eibex
+
+directory = path.dirname(path.realpath(__file__))
+database = sqlite3.connect(f"{directory}/files/reactionlight.db")
+db = database.cursor()
+
+db.execute("CREATE TABLE IF NOT EXISTS 'channels' ('message_id' INT, 'channel' INT);")
+
+embeds_creation = {}
```

Comment:
My bad for that, but maybe "embed" isnt really the right word anymore now :/

Reply:
Also, pretty weird to keep a global variable if you want to uses classes IMO, but that's just personal preferences

Reply:
Yeah, I could use a different word other than embed.

This might sound quite dumb, but how would I access an instance of a class between different steps without a global dict?

Right now to get the instance of the class I'm interested in is by using the key "user_channel" in the dict.

Without a dict wouldn't the bot lose the possibility of managing multiple simultaneous reaction-role messages?